hack/update-bundles.sh

```diff
@@ -55,9 +56,10 @@ function repo_name() {
 
 function conftest_push() {
   if [ $1 == "data" ]; then
-    $DRY_RUN_ECHO conftest push --policy '' --data data $2
+    $DRY_RUN_ECHO ${CONFTEST} push --policy '' --data data $2
   else
-    $DRY_RUN_ECHO conftest push --policy policy $2
+    pwd
```

Comment:
Intended?

Reply:
I was troubleshooting go modules, one can't point to a `go.mod` outside of the project tree, i.e. in `$DIR/a` one can't do `go run -modfile $DIR/b/go.mod`